src/browser/application.js

```diff
@@ -1,5 +1,5 @@
 import app from 'app';
```

Comment:
For forward compatibility, you should update all electron modules to be off the `require('electron')` import.

See: https://github.com/atom/electron/releases/tag/v0.35.0